docs/user/tutorials/quickstart.md

```diff
@@ -0,0 +1,138 @@
+# Quickstart
+
+This guide covers how you can quickly get started using Hypper.
+
+If you already know Helm, this will be a breeze, as Hypper follows Helm's UI,
+being a drop-in replacement, and extends its functionalities to install Helm
+and Hypper charts alike.
+
+## Prerequisites
+
+The following prerequisites are required for a successful and properly secured
+use of Hypper.
+
+1. A Kubernetes cluster.
+1. Deciding what security configurations to apply to your installation, if any
+1. Installing and configuring Hypper.
+
+
+## Install Kubernetes or have access to a cluster
+- You must have Kubernetes installed. We recommend the latest stable release of
+  Kubernetes.
+- You should have a local configured copy of `kubectl`, to check results from
+  the tutorials.
+
+## Install Hypper
+
+See the [installation guide](docs/user/tutorials/installing.md).
+
+
+## Configuring Hypper by initializing repositories
+
+Hypper enables you to deploy Helm and Hypper charts into your Kubernetes
+cluster. To simplify that, the best way is to add a chart repository, for
+example one with Helm charts:
+
+```terminal
+$ hypper repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+You can add several repositories. Let's add also a repository containig charts
+with some Hypper functionality:
+
+```terminal
+$ hypper repo add rancher-charts https://charts.rancher.io
+```
+
+Now, you can list the repository that Hypper can access:
```

Comment:
should we mention `that hypper can install charts from` instead to make it clearer?